doc/write_your_own/components/DimPicker1.ipynb

```diff
@@ -12,47 +23,65 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load cube."
+   ]
+  },
+  {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": false
+   },
    "outputs": [],
    "source": [
     "cube = iris.load_cube(iris.sample_data_path('A1B.2098.pp'))\n",
     "print cube"
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compose list of options and then construct widget to present them, along with a default option.  Display the widget using the IPython display call."
+   ]
+  },
+  {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": false
+   },
    "outputs": [],
    "source": [
     "coordinates = [(coord.name()) for coord in cube.coords()]\n",
     "\n",
     "dim_x = ipywidgets.ToggleButtons(\n",
     "    description='Dimension:',\n",
     "    options=coordinates,\n",
-    "    value = 'time')\n",
+    "    value='time')\n",
     "\n",
     "IPython.display.display(dim_x)"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "print dim_x.value"
+    "Print selected widget value for clarity."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": false
+   },
    "outputs": [],
    "source": [
-    ""
+    "print dim_x.value "
```

Comment:
In all three dim_picker notebooks there is that single whitespace element at the end of the last cell.